src/declarations/interfaces/MotionActivityEvent.d.ts

```diff
@@ -22,7 +22,7 @@ declare module "react-native-background-geolocation" {
     * | `on_bicycle`   |
     * | `in_vehicle`   |
     */
-    activity: string;
+    type: string;
```

Comment:
```suggestion
    activity: 'still' | 'walking' | 'on_foot' | 'running' | 'on_bicycle' | 'in_vehicle;
```
anyway, it should remain `activity` and shoukd **NOT** be changed to `type`
@christocracy